packages/theme-dark-extension/style/variables.css

```diff
@@ -125,6 +125,13 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-ui-font-color1: rgba(255, 255, 255, 0.87);
   --jp-ui-font-color2: rgba(255, 255, 255, 0.54);
   --jp-ui-font-color3: rgba(255, 255, 255, 0.38);
+  --jp-ui-font-color4: rgba(255, 255, 255, 0.24);
+  --jp-ui-font-color5: rgba(255, 255, 255, 0.12);
```

Comment:
Are those used anywhere?

Reply:
Sorry for the late answer. After retesting this PR, the conclusion is that the visual result is not good enough and a bit counter intuitive in some cases with the inactive elements more visible than active ones, like for instance on this screen shot (elements in dark blue grey are inactive ones):

![Screenshot from 2023-01-11 17-01-19](https://user-images.githubusercontent.com/99649086/211855043-125437ae-65ef-4939-af14-0d3aa76e54bf.png).

So I propose to close this PR.